app/javascript/channels/posts_channel.js

```diff
@@ -0,0 +1,23 @@
+// import consumer from "./consumer"
```

Comment:
we have an entire file here that is commented out. why?

Reply:
thanks, ill remove this